### DISCUSSION TITLE
Manage Repo Access does not show Collaborators and Team Member, and not sending email invites

### DISCUSSION THREAD

**Discussion by Bernard-Foster** [community/community#112215] [Repositories] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

When giving Repo access to collaborators and teams , the Manage Access pages does not show any team members or collaborators as of this morning.  
3 / 12/ 24

For me, It only shows Repo members for 5 milliseconds before it 'auto-wipes' them away. 

Sending Repo invites via email seems to fail for me but if I add in a team it works but the UI still does not reflect in the settings page.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by moozunch** [comment]
Hi, try to ask your friend you invite as the collaborator to open the repo and accept the invitation there, hope this help ^____^

**Comment by kuciapakrystian** [comment]
Here are a few steps you can take to troubleshoot and potentially resolve the issue:

Refresh the Page: Sometimes, UI glitches can be resolved by simply refreshing the page. Try refreshing the Manage Access page to see if the repository members and collaborators reappear.

Clear Browser Cache: Cached data in your browser can sometimes cause unexpected behavior. Try clearing your browser cache and reloading the page to see if that resolves the issue.

Check Network Connection: Ensure that your internet connection is stable and there are no network issues that could be affecting the loading of data on the page.

Try Different Browsers or Devices: If the issue persists, try accessing the Manage Access page from a different web browser or device to see if the problem is specific to one environment.

Contact Support: If none of the above steps resolve the issue, consider reaching out to the support team of the platform you're using (e.g., GitHub support) for further assistance. They may be able to provide insights or help troubleshoot the issue on their end.

Monitor Platform Status: Sometimes, platform-wide issues or maintenance activities can affect specific functionalities. Check the status page or official announcements from the platform to see if there are any known issues affecting access management features.

Documentation and Community Forums: Check the documentation and community forums of the platform you're using. Other users might have experienced similar issues and might have found workarounds or solutions.

If the problem persists despite these efforts, it's best to report the issue to the platform's support team for further investigation and resolution.

**Reply by ghostinhershell** [reply]
Hi @kuciapakrystian, 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by Bernard-Foster** [comment, marked as answer]
The text field for 'Find Team, Organization Members ' auto-populates with my own email, which after a second filters other team members from the ui.

 After clearing the textfield of my own email they showed up. ;-)

**Comment by queenofcorgis** [comment]
@Bernard-Foster thanks for coming back and letting us know what was going on. 

For any future posts:  You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you to our Repos category. Cheers!
